examples/shopify/README.md

```diff
@@ -24,6 +24,10 @@ This example includes a home page for listing products by category and a product
 
    Then head over to [Shopify](https://www.shopify.com/) and create a store. Here is [a good starting point](https://www.shopify.com/online) to get you started.
 
+   Note: This custom storefront example assumes all products are in stock, so make sure all products have "Track quantity" unchecked as pictured below.
+
+      <img width="600" src="https://user-images.githubusercontent.com/20950876/187238713-22fd0c65-8d9b-4eab-b94e-1e498053f270.png">
+
```

Comment:
other edits to this file are from prettier

Reply:
For future PRs, it would be great if you could separate `prettier --write .` runs in their own commit. It allows the reviewer to avoid that noise and focus on relevant changes 😄